test/k8sT/Istio.go

```diff
@@ -0,0 +1,171 @@
+// Copyright 2017 Authors of Cilium
```

Comment:
The commit message for this is way too brief. In the final version, please be sure to detail what the test framework does, including topology of the setup, tests that have been added along with brief descriptions of what they do, etc.